docs/INSTALL.md

```diff
@@ -49,7 +49,8 @@ tool installations are described in [Advanced](#Advanced).
 **Installing F´ Python Packages**
 
 ```
-pip install fprime/
+pip install -U setuptools wheel pip
+pip install -U -r fprime/requirements.txt
```

Comment:
Any particular reason to use the `-U` flag here? If a python dependency makes a breaking change, I feel like that kind of defeats the point of using a `requirements.txt` for reproducible builds.

Reply:
I guess the `-U` flag is ignored if a specific version is specified pip install -U flag ignored if specific version is specified

Reply:
Yes, here we are using the -U flag to force the version to that specified.  The use case being: you've install a previous revision's requirements.txt and need to upgrade/downgrade.  The -U was intended to force a change from what you have to what is needed.

Perhaps this is not necessary. I will test.